<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StationChoiceBox</class>
 <widget class="QWidget" name="StationChoiceBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 50);</string>
  </property>
  <widget class="QPushButton" name="PB_Station_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>1</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>余杭高铁站</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Station_2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>2</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>许村站</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Station_8">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>130</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>8</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>桐九公路站</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Station_10">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>210</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>10</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>皮革城站</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Station_7">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>7</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>盐官站</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Station_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>5</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>长安东站</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Station_12">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>210</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>12</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
font:12px;</string>
   </property>
   <property name="text">
    <string>浙大国际校区站</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Station_3">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>3</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>海宁高铁西站</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Station_11">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>210</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>11</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>海昌路站</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Station_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>4</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>长安站</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Station_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>6</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>周王庙站</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Station_9">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>9</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>斜桥站</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Cancel">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>270</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>13</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>555</width>
     <height>328</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50,50,100);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>lbl_bg</zorder>
  <zorder>PB_Station_1</zorder>
  <zorder>PB_Station_2</zorder>
  <zorder>PB_Station_8</zorder>
  <zorder>PB_Station_10</zorder>
  <zorder>PB_Station_7</zorder>
  <zorder>PB_Station_5</zorder>
  <zorder>PB_Station_12</zorder>
  <zorder>PB_Station_3</zorder>
  <zorder>PB_Station_11</zorder>
  <zorder>PB_Station_4</zorder>
  <zorder>PB_Station_6</zorder>
  <zorder>PB_Station_9</zorder>
  <zorder>PB_Cancel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
